### PR TITLE
Make the PHP scripts appear in Visual Studio's Solution Explorer

### PR DESCRIPTION
**Pull request by Indigo744**
This is simple QoL improvements to be able to see the PHP test files from within the VS Solution Explorer:
image

### CODE REVIEW

**Comment by Indigo744** [comment]
Damn I didn't expect a simple change like this would trigger an error...

**Comment by jakubmisek** [comment]
seems it's a fail of a previous commit

**Comment by jakubmisek** [comment]
fixed in https://github.com/peachpiecompiler/peachpie/commit/3937ae622d3b3b8fb7d159697d4900af82019902

File: src/Tests/Peachpie.ScriptTests/Peachpie.ScriptTests.csproj [hunk 1]
```diff
@@ -26,6 +26,13 @@
     <ProjectReference Include="..\..\Peachpie.Runtime\Peachpie.Runtime.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- Make the PHP scripts appear in Visual Studio's Solution Explorer -->
+    <None Include="..\..\..\tests\**">
+      <Link>Scripts\%(RecursiveDir)%(Filename)%(Extension)</Link>
```
Comment (jakubmisek) [inline src/Tests/Peachpie.ScriptTests/Peachpie.ScriptTests.csproj:32]: can we change it from `Scripts\` to `tests\` so it is consistent with directory structure

Comment (jakubmisek) [inline src/Tests/Peachpie.ScriptTests/Peachpie.ScriptTests.csproj:32]: maybe use full path and *.php pattern so there are just test files
```xml
<None Include="$(MSBuildThisFileDirectory)..\..\..\tests\**\*.php">
```

**Review by jakubmisek — CHANGES_REQUESTED** [review]
thanks! I'd just ask for some minor changes there

**Comment by Indigo744** [comment]
Changes implemented 😸

**Pull request merged**